roles/nsx/tasks/main.yml

```diff
@@ -1,159 +1,140 @@
 ---
-- name: NSX | Run kube proxy
-  run_once: true
-  import_role:
-    name: kube_proxy_and_dns
-
-- name: NSX | Ensure nsx-system project exists
-  oc_project:
-    name: nsx-system
-    state: present
-    node_selector:
+- when: inventory_hostname == groups.oo_first_master.0
```

Comment:
It would be best to rewrite playbooks/openshift-master/private/config.yml to run this role on `oo_first_master` instead

Reply:
it makes sense but if I move it in a separate play in order to provide host: oo_first_master I am losing the restart master handler. I will just add inventory_hostname == groups.oo_first_master.0 in the when clause in playbooks/openshift-master/private/config.yml as oo_first_master is undefined by its own as a var.

Reply:
I see, okay, makes sense